common/app/contentapi/ContentApiClient.scala

```diff
@@ -104,6 +104,11 @@ trait MonitoredContentApiClientLogic extends CapiContentApiClient with ApiQueryD
 
   val httpClient: HttpClient
 
+  override implicit val executor = ScheduledExecutor()
+  val retryDuration = Duration(250L, TimeUnit.MILLISECONDS)
+  val retryAttempts = 5
```

Comment:
We may need to adjust this ☝️to keep it within the expectations of the circuitbreakers